tools/travis/splits/split_misc.sh

```diff
@@ -72,5 +72,8 @@ run_test "SQL Client end-to-end test for Kafka 0.10" "$END_TO_END_DIR/test-scrip
 run_test "SQL Client end-to-end test for Kafka 0.11" "$END_TO_END_DIR/test-scripts/test_sql_client_kafka011.sh"
 run_test "SQL Client end-to-end test for modern Kafka" "$END_TO_END_DIR/test-scripts/test_sql_client_kafka.sh"
 
+run_test "Shaded Hadoop S3A with credentials provider end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wordcount.sh hadoop_with_provider"
+run_test "Shaded Presto S3 with credentials provider end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wordcount.sh presto_with_provider"
```

Comment:
Afaik, the `run-nightly-tests.sh` is not used in CI runs, but it's mentioned in the `README.md`.
I think it would be nice to add those new tests into the `run-nightly-tests.sh` too, to keep the file in sync with tests splits.

Reply:
The `run-nightly-tests.sh` should actually be removed entirely. It's already out-of-sync with the actually run tests, and acts as a trap for committers who aren't that familiar with the Travis setup. Multiple tests are defined in there that are missing from the splits, and this is likely not intentional.

Reply:
If anything it should be reworked to call into all splits.